scripts/main.py

```diff
@@ -35,9 +35,15 @@ def _request_headers(self):
 
     def _comment_screenshots(self, images):
         """Comments Screenshots to the pull request"""
-        string_data = "## Here are the Screenshots after the Latest Changes\n\n"
+        run_url = (
+            f"https://github.com/{self.configuration.GITHUB_REPOSITORY}/"
+            f"actions/runs/{self.configuration.GITHUB_RUN_ID}"
+        )
+
+        string_data = f"{self.configuration.CUSTOM_ATTACHMENT_MSG} `{self.configuration.GITHUB_SHA}`. \
+            _You can inspect the workflow run at {run_url}_. \n\n"
```

Comment:
URL as link instead of plain text url (`You can inspect the workflow run at {run_url}` -> ``You can inspect the workflow run [here]({run_url})`